src/net_processing.cpp

```diff
@@ -2267,6 +2355,29 @@ static void ProcessGetCFCheckPt(CNode& peer, CDataStream& vRecv, const CChainPar
     connman.PushMessage(&peer, std::move(msg));
 }
 
+/**
+ * Announce transactions a peer is missing after reconciliation is done.
+ * No need to add transactions to peer's filter or do checks
+ * because it was already done when adding to the reconciliation set.
+ */
+void static AnnounceTxs(const std::vector<uint256>& remote_missing_wtxids, CNode& pto, const CNetMsgMaker& msgMaker, CConnman& connman)
```

Comment:
It's easy enough to instantiate a new `CNetMsgMaker` from the CNode, rather than passing it as an argument. If you make this function a member of `PeerManagerImpl` you can also avoid passing the `CConnman&`.